.gitignore

```diff
@@ -1,5 +1,9 @@
 bin/
-dist/ 
+dist/
+nohup.out
+
+# Temporary
+umpleonline/index.php
```

Comment:
For "working" in my docker container, I need index.php (our default is umple.php which is then configured through apache / nginx to serve as the default.  I might do a separate PR with both, and then a third to remove it, once we see things working just fine.